### ISSUE TITLE
Feature Request: Setting "Disable icons in source mode"

### ISSUE THREAD

**Issue by DirkBloedorn** [mgmeyers/obsidian-list-callouts#35]
Great plug-in!
I work almost exclusively in preview mode and rarely switch to source mode when I want more control.
I use the plugin mostly with tags (#task, #goal) along with the Tasker plugin.

In source mode it would be good to have "#task" etc showed up instead of the icon - or if I don't remember "&" was the character for a bookmark.

For this a setting would be handy: "Disable icons in source mode".

**Comment by MSBack** [comment]
I agree!

**Comment by mindfad** [comment]
Bumping! This would be very helpful if the parsing of list callouts into icons could be turned off when in source mode. 🙂 Thanks!

**Comment by ltroj** [comment]
I agree, source mode should be plain text only!

**Comment by churnish** [comment]
Any custom styling added by the plugin should not be displayed in Source mode 👍

**Comment by kazerniel** [comment]
Yes please! It would let me edit my "tag icons" without having to open the file in an external editor.

**Comment by danyim** [comment]
@DirkBloedorn @FeralFlora @mindfad @ltroj @churnish @kazerniel This is in my maintained fork, [List Callouts, Improved](https://github.com/danyim/obsidian-list-callouts-improved), as of [1.6.4](https://github.com/danyim/obsidian-list-callouts-improved/releases/tag/1.6.4), released today. It is in the community plugin catalog under that name, and it has an import button for your existing List Callouts settings.

Source mode now shows the raw markdown, `- & text` and so on, with nothing drawn over it: no icon, no marker, no background band, and no highlight color. The callout character is there to see and edit, and switching between source mode and Live Preview redraws on its own. Live Preview and reading view are unchanged.

![The README's first note in source mode: every list item shows its plain callout character with no plugin styling, in light and dark mode](https://raw.githubusercontent.com/danyim/obsidian-list-callouts-improved/main/screenshots/callout-characters.source.png)

It is the behavior rather than a setting, since everyone here asked for the same thing and it matches how Obsidian treats source mode. If you want it as a toggle instead, open an issue over there.
